great_expectations/execution_engine/sparkdf_execution_engine.py

```diff
@@ -480,6 +493,29 @@ def get_domain_records(
 
         return data
 
+    def _combine_row_conditions(
```

Comment:
@anthonyburdi is this method static?

Reply:
Effectively yes, but it's called only within the context of the instance so I've left it an instance method.

Reply:
@anthonyburdi Doesn't PyCharm give us warnings?  Perhaps we can turn it off, because the next person will almost certainly add the "@staticmethod" decorator, thereby contravening your rationale.  I feel like something has to be done.  Thanks.